src/gam/gapi/__init__.py

```diff
@@ -319,11 +319,12 @@ def get_all_pages(service,
   Returns:
     A list of all items received from all paged responses.
   """
+    if page_args_in_body:
+        kwargs.setdefault('body', {})
```

Comment:
Does the same thing in two lines that my code did in 1?

Reply:
This is to ensure that body gets set in kwargs in case _get_max_page_size_for_api_call returns None. Notice that _get_max_page_size_for_api_call always returns {'maxResults': max_results} when it should return {'pageSize': max_results} for Chrome Policies.